private/signer/v4/v4.go

```diff
@@ -116,6 +116,13 @@ type signer struct {
 	signedHeaderVals http.Header
 }
 
+// @readonly
+var emptySha256Hex string
+
+func init() {
+	emptySha256Hex = hex.EncodeToString(makeSha256([]byte{}))
+}
```

Comment:
This could be just a `const` since the value won't change.

```go
// EmptyStringSHA256 is a SHA256 of an empty string
const EmptyStringSHA256 = `e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855`
```